frontend/app/forms/inputs/text-area.component.js

```diff
@@ -65,4 +65,8 @@ const css = `
     display: flex;
     flex-direction: column;
   }
+
+  & textarea {
```

Comment:
rather than selecting `textarea` element, I think doing a class selector may be better practice, perhaps `.text-area`?